apps/condo/domains/billing/utils/countrySpecificQRCodeParsers/ru.qrCodeParser.js

```diff
@@ -14,26 +13,26 @@ const { get } = require('lodash')
 
 /**
  * @param {string} qrStr The QR code string got from the picture
+ * @param {function} logger ValidateQRCodeService logger
  * @return {TRUQRCodeFields}
  */
-function parseRUReceiptQRCode (qrStr) {
-    const matches = /^ST(?<version>\d{4})(?<encodingTag>\d)\|(?<requisitesStr>.*)$/g.exec(qrStr)
+function parseRUReceiptQRCode (qrStr, logger) {
```

Comment:
imho this is a bad idea to make this function dependent of some external logger. Look like as bad developer experiance, it's will be hard to test this function. Maybe it's better write logs in the place where this function called?

Reply:
rm it